### PR TITLE
More miscellaneous matches

### PR DESCRIPTION
**Pull request by Tmcg2**
And more mips_to_c refreshes

Signed-off-by: Taggerung <tyler.taggerung@gmail.com>

### CODE REVIEW

**Comment by MegaMech** [comment]
Does the refreshing ignore `#define NON_MATCHING` for funcs that are already close to matching?

**Review by MegaMech — DISMISSED** [review]

**Comment by Tmcg2** [comment]
I do the refreshing by hand and in general I try to leave NON_MATCHING functions alone. No need to get rid of an existing attempt if I can't come up with something better (or, at least, that I think is better)

File: src/crash_screen.c [hunk 1]
```diff
@@ -83,120 +83,38 @@ void crash_screen_draw_glyph(u16 *framebuffer, s32 x, s32 y, s32 glyph) {
...
+    s32 row;
+    s32 column;
+
+    for (var_s0 = 0x2E, var_s2 = 0x28; var_s0 > 0x2C; var_s0--, var_s2++) {
+        for (row = var_s2; row < var_s0; row++) {
+            for (column = var_s2; column < var_s0; column++) {
+                if ((var_s0 - var_s2) == 6) {
+                    framebuffer[(row * 320) + column] = RED_COLOUR;
+                } else {
+                    framebuffer[(row * 320) + column] = WHITE_COLOUR;
```
Comment (MegaMech) [inline src/crash_screen.c:113]: Can you update the func to align with our latest diff:
(change func prototype and `framebuffer[row * 320 + column] =`
https://decomp.me/scratch/pUilB

and then I'll merge.

**Review by MegaMech — CHANGES_REQUESTED** [review]

File: src/crash_screen.c (hunk 1 above)
Comment (Tmcg2) [reply src/crash_screen.c:113]: I think its currently up-to-date with that? Prototype seems right and the array access is as you commented.

Comment (MegaMech) [reply src/crash_screen.c:113]: Oh, my bad! I didn't look that closely apparently.

**Pull request merged**